lua/ge/extensions/MPVehicleGE.lua

```diff
@@ -845,8 +847,16 @@ local function applyVehSpawn(event)
 
 	nextSpawnIsRemote = true -- this flag is used to indicate whether the next spawn is remote or not
 
-	if settings.getValue("simplifyRemoteVehicles") and simplified_vehicles[vehicleName] then
-		vehicleConfig.parts[vehicleName..'_body'] = simplified_vehicles[vehicleName]
+	if settings.getValue("simplifyRemoteVehicles") then
+		local expectedPartID=simplified_vehicles[vehicleName]
+		if expectedPartID then
+			local ioCtx={preloadedDirs={string.format("/vehicles/%s/",vehicleName),"/vehicles/common/"}} -- Fake io context for jbeamIO
+			local slotMap=jbeamIO.getAvailableSlotMap(ioCtx) -- slots2 compatible, maybe theres better way then require jbeamio and let it load the files again...
```

Comment:
```suggestion
			local slotMap = jbeamIO.getAvailableSlotMap(ioCtx) -- slots2 compatible, maybe theres better way then require jbeamio and let it load the files again...
```
Please leave whitespace around assignment statements. Do this everywhere in your code

Reply:
done